modules/configure-signature-integration.adoc

```diff
@@ -5,20 +5,46 @@
 [id="configure-signature-integration_{context}"]
 = Configuring signature integration
 
-Before performing image signature verification, you must first add your Cosign public keys in {product-title-short}.
+Before performing image signature verification, you must first create a signature integration in {product-title-short}.
+
+A signature integration can be configured with multiple verification methods. The following verification methods are supported:
+* Cosign public keys
```

Comment:
I think you need a blank line here so the bullets look OK:
![Screenshot 2024-06-27 at 1 48 17 PM](https://github.com/openshift/openshift-docs/assets/96192119/a2356549-4789-4110-a6e1-736256a72a91)